.github/workflows/continuous-benchmarking.yml

```diff
@@ -0,0 +1,108 @@
+# MIT License
+#
+# Copyright (c) 2022 Dilina Dehigama and EASE Lab
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+name: Run scheduled experiments
+
+on: 
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
I guess running on PRs and push-es wouldn't hurt. 

in future, please request reviews from @amariucaitheodor, he is the main maintainer.

Reply:
Sure. But I guess once other experiments are also added, it would take quite a long time (few hours) to run the whole workflow. Since even now it takes ~2.5 hours to run with just two experiments. Would that be okay? 
And sure thing will request reviews from @amariucaitheodor in the future.